db.js

```diff
@@ -531,6 +557,7 @@ function upgrade (db) {
     autoIncrement: false
   })
   notes.createIndex(ATTRIBUTED_TO_FIELD, ATTRIBUTED_TO_FIELD, { unique: false })
+  notes.createIndex(IN_REPLY_TO_FIELD, IN_REPLY_TO_FIELD, { unique: false })
```

Comment:
We need to bump the db version and only run this code when the version number is smaller than the latest.

https://www.mozzlog.com/blog/indexeddb-migration-using-idb